front-end/src/security_privacy/rsa/encryptor.ts

```diff
@@ -0,0 +1,56 @@
+
+var rsa = require("node-rsa");
+//for salt
+var  crypto = require('crypto');
+var forSalt;
+
+//file input
+var fs = require("fs");
+// create new keys variable
+var publicKey = new rsa();
+var privateKey = new rsa();
+
+//retreved data from both .pem files
+var public = fs.readFileSync("./key/public.pem", "utf8");
+var private = fs.readFileSync("./key/private.pem", "utf8");

```

Comment:
This is the front-end (browser). There is no filesystem (fs). I don't think this will work.

Reply:
I see, and yeah I still search for a way store those keys more secure.